### ISSUE TITLE
Learning center not working in 0.9.0

### ISSUE THREAD

**Issue by julesvirallinen** [Visor-Live/visor-builds#12]
In visor 0.9.0 on mac the Learning center crashes when changing tabs. 

![Kapture 2021-11-05 at 18 24 33](https://user-images.githubusercontent.com/4622905/140544710-9c3db52e-e575-42aa-9d5f-c04adf0d4ba4.gif)

Trace:

```

TypeError: Cannot read property 'current' of null
    at file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:1:60518
    at ra (file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:35:83207)
    at oa (file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:35:83310)
    at fl (file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:35:101244)
    at t.unstable_runWithPriority (file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:43:3844)
    at zr (file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:35:45022)
    at dl (file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:35:97716)
    at Ja (file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:35:93870)
    at file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:35:45313
    at t.unstable_runWithPriority (file:///private/var/folders/b3/42tkmpw90r563f5t5b5_7fbw0000gn/T/AppTranslocation/8F79704E-972C-4E30-B9A8-58C95F307033/d/Visor.app/Contents/Resources/app/build/learn.js:43:3844)

    in ForwardRef
    in div
    in Unknown
    in div
    in div
    in Unknown
    in div
    in div
    in div
    in div
    in vo
    in v
    in div
    in Hr
    in Wr
    in div
    in z
    in s
    in p
    in Unknown
```

**Comment by EmperorJack** [comment]
Thanks for reporting this @julesvirallinen - I have a fix staged for an upcoming release 🐛

**Comment by EmperorJack** [comment]
This should now be fixed in version 0.10.0, please re-open if still an issue.
